### DISCUSSION TITLE
3번째 미팅 때 추가로 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#8] [Q&A] [answered]
1. PR 리뷰에 관해서
    - 프로세스
        1. 작업을 우선 다른 브랜치에서 진행
        2. 리뷰가 필요한 코드만 먼저 Commit을 하고 PR을 작성
        3. 나머지 작성된 코드들은 Commit 후 Merge
    - 위과 같은 프로세스로 진행하면 되는건지?
    - 모든 작업을 일단 타 브랜치에서 하고 PR을 통해 Merge하는 프로세스를 구축하는 방식?
    - Merge가 되기 전에는 해당 소스는 수정이 불가능?
    - PR에서 특정 코드만 아래와 같이 보여주는 방법 한번만 더 알려주세요!
    ![source](https://user-images.githubusercontent.com/75019048/107333283-976f0080-6af8-11eb-8a5e-f63a494ff3c3.PNG)
    - 같은 브랜치에서 계속 수정하고 커밋을 하면 자동으로 기존 PR에 등록이 되는 현상? 
2. 매주 토요일에 하는 스터디 링크 다시 안내 좀 해주세요..ㅠ

3. Discussion에 기록을 남기면 참조는 어떻게 하면 좋을지?
    - ex) @로 참조하기?
    
4. 향후 진로에 대해서..
    - 닷넷 개발자?(프론트엔드 or 백엔드)
    - 유니티 개발자?(단순 AR 개발만)
    - 개발자로서의 기본 역량과는 별개(이 부분은 어느정도 고민이 끝남)
    - 단순히 직업적으로 어떤 길을 가면 좋을지에 대한 고민들

**Comment by jongfeel** [comment, marked as answer]
1. PR 작성후 commit

이렇게 하면 commit이 계속 누적됩니다. 논의한 내용을 바탕으로 계속해서 업데이트를 하고 싶을 때 하면 유용합니다.
물론 reviewer가 suggession을 통해 변경을 할 수 있고 이 내용을 commit 하면
pull request 요청자가 별도로 commit->push를 하지 않아도 github 상에서 진행 가능해요.

코드 라인을 드래그 하면 multi line select가 가능합니다. 한번 보여드렸는데, 기억이 안나면 또 얘기해주세요.

2. https://github.com/ThinkAboutSoftware/OnlineSelfCodingGroup

3. mention으로 하면 참조가 잘 되는 것 같습니다.

4. 유니티 개발에 대한 이해도를 높이고 더불어서 C# 관련된 객체지향 방법론을 익힌다면, 다른 분야로 이직해도 어느 정도 적응이 가능할 것 같아요.
